src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/mockserver/WorkforceAllocationsToDeliusMockServer.kt

```diff
@@ -158,6 +158,24 @@ class WorkforceAllocationsToDeliusMockServer : ClientAndServer(MOCKSERVER_PORT)
     )
   }
 
+  fun setGetApopUsers() {
```

Comment:
This method is almost identical to  fun setApopUsers()  but with a different staffCode. Can we reuse setApopUsers and parametise the staffcode.

Reply:
Missed your method! Can do away with mine and just use setApopUsers